values.yaml

```diff
@@ -15,7 +15,7 @@ docker:
 ##
 cog:
   name: cog
-  serviceType: ClusterIP
+  serviceType: NodePort
```

Comment:
Shouldn't Cog's serviceType be `ClusterIP` if we're setting an nginx service in front of it?